pyproject.toml

```diff
@@ -74,6 +76,7 @@ rfc3339 = "^6.2"
 typing = "^3.7.4"
 item-synchronizer = "^1.1.4"
 bubop = { version = "0.1.10", allow-prereleases = true }
+google-auth-httplib2 = "^0.1.0"
```

Comment:
Is this required for all the integrations? If it is google-dependent could you mark it as optional and add it to the `google` extra [here](https://github.com/bergercookie/syncall/pull/79/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R82)?

Reply:
done